atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/KvsMigrationCommand.java

```diff
@@ -126,6 +126,8 @@ public int execute(AtlasDbServices fromServices, AtlasDbServices toServices) {
         }
         if (migrate) {
             migrator.migrate();
+
+            migrator.cleanup();
```

Comment:
You can't run it online trivially as it uses many different transactions to complete the reads, which would result in an inconsistent view of the DB in the migrated service.  Ei. writes that had done into an already scanned table would be lost or maintained if they occurred on table that had yet to be scanned.